### ISSUE TITLE
Core dumped Error while run Appimage

### ISSUE THREAD

**Issue by ssmns** [toolstack/Folio#104]
When I open Folio.appimage, the below error occurred:
```
(com.toolstack.Folio:577910): GLib-GIO-ERROR **: 16:39:29.438: Settings schema 'com.toolstack.Folio.Theme' is not installed
[1]    577910 trace trap (core dumped)  ./Folio
```
Folio: Version 24.10
Os: ubuntu 23.10

**Comment by toolstack** [comment]
Interesting, will take a look, the AppImage runs on my dev box but it has the various parts of Folio installed locally so it may be finding them regardless or what's in the AppImage.

**Comment by ngdangtu-vn** [comment]
I think I have a similar problem but more cirtical  #136

**Comment by toolstack** [comment]
This should now be resolved via #191
